lib/session.js

```diff
@@ -100,6 +99,12 @@ function autoStoreData (req, res, next) {
   if (!req.session.data) {
     req.session.data = {}
   }
+  res.locals.data = {}
+
+  if (req.originalUrl.startsWith('/manage-prototype/')) {
```

Comment:
might it be better on line 99, so autoStoreData really does nothing at all?

Reply:
I had that to start with but then downstream things failed because `data` didn't exist.

Reply:
ah cool